src/js/components/__tests__/MarathonTaskDetailsList-test.js

```diff
@@ -1,106 +0,0 @@
-jest.dontMock('../MarathonTaskDetailsList');
```

Comment:
This test file only tested a method that has been abstracted into multiple parts. I tested the new component in `TaskEndpointsList-test`